daml-lf/encoder/src/main/scala/com/digitalasset/daml/lf/archive/testing/EncodeV1.scala

```diff
@@ -141,13 +141,17 @@ private[digitalasset] class EncodeV1(val minor: LV.Minor) {
     @inline
     private implicit def encodeTypeBinder(binder: (String, Kind)): PLF.TypeVarWithKind = {
       val (varName, kind) = binder
-      PLF.TypeVarWithKind.newBuilder().setVar(varName).setKind(kind).build()
+      PLF.TypeVarWithKind
+        .newBuilder()
+        .setName(varName)
+        .setKind(kind)
+        .build()
     }
 
     @inline
     private implicit def encodeFieldWithType(nameWithType: (String, Type)): PLF.FieldWithType = {
       val (name, typ) = nameWithType
-      PLF.FieldWithType.newBuilder().setField(name).setType(typ).build()
+      PLF.FieldWithType.newBuilder().setName(name).setType(typ).build()
```

Comment:
heads up here that this is non-interned and we might want to change that if we don't support both schemes in v.dev.

Reply:
I will fix that in another PR. this one is already too big.